rfcs/44-removed-filter.md

```diff
@@ -0,0 +1,87 @@
+# Feature Name: `removed-filter`
+
+## Summary
+
+`Removed<C>` can be used as a query filter, with reliable change detection behavior.
+Users must opt-in to enabling this by configuring the `Component` trait for `C`.
+
+## Motivation
+
+Removal detection is a powerful tool for ensuring the validity of game state, but the existing `RemovedComponents` system parameter is unwieldy (see the motivating [bevy #2148](https://github.com/bevyengine/bevy/issues/2148)).
+
+It cannot be easily composed with other queries, has an unexpected API, and only persists for two frames, unlike reliable change detection.

```

Comment:
Should note that it doesn't actually persist for two frames which makes it more unwieldy.